WcaOnRails/app/views/competitions/_nearby_competitions.html.erb

```diff
@@ -64,6 +111,10 @@
 <script>
   (function() {
     $('[data-toggle="tooltip"]').tooltip();
+    $('[data-toggle="popover"]').popover({
+      content: $('#popover-content').html(),
```

Comment:
Won't this affect *every* popover on the page? This doesn't seem right. Can you pass content and html via `data-content` and `data-html` instead?